main/templates/main/login.html

```diff
@@ -2,34 +2,49 @@
 {% load static %}
 <!DOCTYPE html>
 <html>
-    <head>
-         <!-- Bootstrap core CSS -->
-        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
-        <title>roo.me - Login!</title>
-    </head>
-    <style>
-        body { background-image: url(../../static/img/roomates-login.jpg);
-        background-repeat: no-repeat;
-        background-size: cover;
-        }
-    </style>
-    <body>
-        <div style="position:fixed; left:45px; top:10px;">
+
+<head>
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}">
+    <title>roo.me - Login!</title>
+</head>
+
+<body>
+
+    <section>
+        {% if messages %}
+        {% for message in messages %}
+        <div class="alert alert-{{message.tags}} alert-dismissible fade show" role="alert">
+            {{message}} <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+        </div>
+        {% endfor %}
+        {% endif %}
+
+        <div class="text-box">
             <h1>Login now!</h1>
-            <div class="card" style="background: rgba(0,0,0,.3); height: 350px; width: 300px;">
+            <div class="card" style="color: rgb(0, 0, 0); background: rgba(252, 252, 252, 0.3); height: 350px; width: 300px;">
                 <div class="card-body">
                     <form method="POST">
                         {% csrf_token %}
                         <fieldset class="form-group">
                             {{form|crispy}}
                         </fieldset>
-                    <div class="form-group">
-                        <button class="btn btn-primary mt-3" type="submit">Lets Go!</button>
-                    </div>
-                    
+                        <div class="form-group">
+                            <button class="btn btn-primary mt-3" type="submit">Lets Go!</button>
+                            <p class="float-right mt-3"><a href="{% url 'register' %}" style="color: rgb(39, 20, 167);">I don't have an account</a></p>
```

Comment:
I think the buttons should be more formal and standard.
**`sign in`.
`register`.**
@Micha546